hips/hip-9999.md

```diff
@@ -0,0 +1,32 @@
+---
+hip: 9999
+title: "Helm Resolving Dependencies First Option"
+authors: [ "protosam" ]
+created: "2021-09-27"
+type: "feature"
+status: "draft"
+---
+
+## Abstract
+All chart dependencies are rendered with the parent chart before being sent to the cluster. This can prevent parent charts from using CRDs from charts that manage their own CRD life cycle. Resolving and installing dependencies in a specified order can solve this issue.
+
+## Specification
+In the `Chart.yaml` adding `dependenciesFirst: true` would signal helm to install the dependencies before the package and they should be installed in order separately. The default value for `dependenciesFirst` should be `false` which indicates to helm to do current behavior.
+
+## Motivation
+Dependencies aren't very interoperable between repos when CRDs are involved.
+
+As-is, it is possible to:
+* Create multi-phase tamplates that involve using `helm install` followed by `helm upgrade` to work around this problem
```

Comment:
there are a few typos here, but this is not enough information to describe how to "work around" the issue.